scripts/models/token/token.constants.js

```diff
@@ -0,0 +1,27 @@
+/**
+ * Supported token categories.
+ */
+const CATEGORIES = {
+  COLOR: "color",
+};
+
+/**
+ * Specificaions for token serials.
+ */
+const SERIAL_SPECIFICATIONS = {
+  INDENT: 2,
```

Comment:
not quite sure what this constant here is for. what is a SERIAL_SPECIFICATIONS.INDENT representing? like how can the number 2 here be explained?

Reply:
Serial specifications is the "shape" of the serial generated by `new Token({ /* ... */ }).serial`. I'll add a comment around this, and potentially change the name to better reflect the intent.